client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -90,6 +95,8 @@ class SiteTools extends Component {
 		const exportText = translate(
 			'Export content from your site. You own your data — take it anywhere!'
 		);
+		const cloneTitle = translate( 'Clone', { context: 'verb' } );
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.